modules/imaging_browser/jsx/onSort.js

```diff
@@ -0,0 +1,15 @@
+/* exported onSort */
```

Comment:
I'm not crazy about this being a global function and think that the onSort and getFormattedColumn should be grouped together into a class (or container React component), but this is the way the module is already designed, so it should be good for now..